Documentation/cmds-main.txt

```diff
@@ -124,3 +124,8 @@ linknvme:nvme-discover[1]::
 linknvme:nvme-connect[1]::
 	Connect to an NVMe-over-Fabrics subsystem
 
+linknvme:nvme-set-property[1]::
+	Writes and shows NVMe-over-Fabrics controller property
```

Comment:
It's very trivial, but this commit would be nicer if it holds get-property only as commit title does.